### ISSUE TITLE
 md-tooltip bug with md-fab-toolbar

### ISSUE THREAD

**Issue by Alexandre-io** [angular/material#5124]
All tooltips are opened on the same location when mouse is over the fab-toolbar. Example:
http://codepen.io/anon/pen/pjrLog

**Comment by ThomasBurleson** [comment]
*  If the toolbar icons have tooltips then tip shows even if the toolbar is closed - http://codepen.io/team/AngularMaterial/pen/QjxQGW

**Comment by SanderElias** [comment]
Duplicate/variation of #4445

**Comment by ishankhare07** [comment]
@SanderElias the issue with the fab-speed-dial is solved
but not with fab-toolbar -> http://codepen.io/ishankhare07/pen/dMpjXP?editors=1010
the position issue has still not fixed
also i'd like to mention that using `ng-if` only affects the visibility, but **DOES NOT** fix the position

**Comment by nicooga** [comment]
Is there a workaround for this?

**Comment by topherfangio** [comment]
@ishankhare07 @nicooga You are correct, this was not fixed with the speed dial fixes. Re-opening and assigning to myself.

**Comment by rahulchanglani** [comment]
+1

**Comment by harvellm** [comment]
+1

**Comment by adibas03** [comment]
+1

**Comment by leandroruel** [comment]
+1

**Comment by VladimirMishyn** [comment]
+1

**Comment by charlieargue** [comment]
+1

**Comment by Gisonrg** [comment]
+1
doesn't work on iOS both Chrome and Safari

**Comment by WanManolo** [comment]
+1

**Comment by ryanzola** [comment]
+1

**Comment by patoncrispy** [comment]
Yeah would be pretty nice to see this fixed. My toolbar is very 'wordy' and would be great to use icons+tooltips instead.

**Comment by johnrix** [comment]
+1

**Comment by Rahul-Sagore** [comment]
I have a way to fix it. But it is not perfect solution. After checking the Angular-Material's code I found that the tooltip gets initialised based on parent's `pointer-events` css properties, which makes sure the parent has pointer events and not set to `none`.

I was having the same problem, So I fixed it by adding pointer events to the menu's icon and removing it after the page has loaded.

### Steps to fix:
1) Add this css code into your css file (Make sure it override existing css)

```css
md-fab-toolbar md-toolbar.pevents__initial {
      pointer-events: initial;
}
```
2) Add `pevents__initial` class to the `md-toolbar` element, which is inside `md-fab-toolbar`:

```html
    <md-toolbar class='pevents__initial'>
        <md-fab-actions class="md-toolbar-tools">
        </md-fab-actions>
    </md-toolbar>
```
3) Add this javascript code to remove `pevents__initial` class from you above html :

```js
setTimeout(function() {
   $('.pevents__initial').removeClass('pevents__initial');
}, 1000);
```

This is just a hack to make tooltip work inside fab toolbar. Need to read the official code more to find more better solution. But this works for me right now.

**Comment by Splaktar** [comment]
I [updated a similar CodePen](https://codepen.io/Splaktar/pen/jOMLwrJ) from #7380 to AngularJS Material 1.2.2 and this issue is still there.

![Screen Shot 2020-12-21 at 00 54 45](https://user-images.githubusercontent.com/3506071/102744209-44ae6f00-4327-11eb-9141-fdf7a89619d0.png)
